src/components/pages/Offers/OfferItem/OfferItem.jsx

```diff
@@ -29,17 +29,23 @@ const OFFER_STATUS_PROPERTIES = {
 }
 
 const OfferItem = ({
+  activateOffer,
+  deactivateOffer,
   location,
   offer,
+  refreshOffers,
   stocks,
   trackActivateOffer,
   trackDeactivateOffer,
-  updateOffer,
   venue,
 }) => {
   function handleOnDeactivateClick() {
     const { id, isActive } = offer || {}
-    updateOffer(id, !isActive)
+    isActive
+      ? deactivateOffer(id)
+      : activateOffer(id).then(() => {
+        refreshOffers(false)
+      })
```

Comment:
Il ne me semble pas qu'on execute le refresh quand on desactive ici.
```suggestion
   const toggleOfferStatus =  isActive ? deactivateOffer : activateOffer
   toggleOfferStatus(id).then(() => refreshOffers(false))
```

Reply:
On n'en avait pas le besoin pour cette US précise parce-que le comportement était différent lorsque l'on activait / désactivait (appel API différent, l'un change le state, pas l'autre etc)

Maintenant qu'on a implémenté la désactivation (US 4283); le refresh se fait à l'activation et à la désactivation. :)